sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueProcessor.cs

```diff
@@ -128,7 +128,7 @@ internal async Task HandlePoisonMessageAsync(QueueMessage message, CancellationT
         /// <returns></returns>
         protected virtual async Task CopyMessageToPoisonQueueAsync(QueueMessage message, QueueClient poisonQueue, CancellationToken cancellationToken)
         {
-            string msg = string.Format(CultureInfo.InvariantCulture, "Message has reached MaxDequeueCount of {0}. Moving message to queue '{1}'.", QueuesOptions.MaxDequeueCount, poisonQueue.Name);
+            string msg = string.Format(CultureInfo.InvariantCulture, "Message Id '{0}' has reached MaxDequeueCount of {1}. Moving message to queue '{2}'.", message.MessageId, QueuesOptions.MaxDequeueCount, poisonQueue.Name);
```

Comment:
Change looks good. However we should also update the existing test [here](https://github.com/Azure/azure-sdk-for-net/blob/441ae5fedd6498281b8fad3983038147f15528bb/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/tests/QueueProcessorTests.cs#L150) so that it verifies this change.